pydantic/main.py

```diff
@@ -222,6 +222,18 @@ def hash_function(self_: Any) -> int:
 # the `BaseModel` class, since that's defined immediately after the metaclass.
 _is_base_model_class_defined = False
 
+BUILTIN_TYPES: AbstractSet[Type[Any]] = frozenset(
```

Comment:
What about bytes, set and frozenset ... ?

Reply:
It's already possible to encode `frozenset`, 'set' and 'bytes' so no need to add them to this set. This set contains types that can't have custom json encoder.

Maybe it's better to rename this variable 🤔

Reply:
I think we have to rename it, not sure what to right now. I'll think about it.

Reply:
k